### PR TITLE
Fs jul19 yevhen sag - moyo-header solution

### PR DESCRIPTION
**Pull request by eugeneSahach**
* [DEMO LINK](https://yevhen-sahach.github.io/layout_moyo-header/)
* [TEST REPORT LINK](https://yevhen-sahach.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -10,10 +10,41 @@
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <header class="header">
+    <a href="#" class="logo"><img src="images/logo.png" alt="logo"></a>
```
Comment (irenhh) [inline src/index.html]: place `` on the next line and don't forget about placing nested tags on the new line

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
+body {
+  margin: 0;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
```
Comment (irenhh) [inline src/style.css]: you can remove width

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  margin: 0;
+  padding: 0;
+}
+
+.nav-link {
+  position: relative;
+  display: flex;
+  text-align: center;
+  font-family: Roboto;
+  font-style: normal;
```
Comment (irenhh) [inline src/style.css]: `font-style: normal` by default

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-link {
+  position: relative;
+  display: flex;
+  text-align: center;
```
Comment (danheim) [inline src/style.css]: Not working in flex element

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  font-family: Roboto;
```
Comment (danheim) [inline src/style.css]: Which family will be displaying if google fonts not responding?
Please, add additional common families like `Arial` or `sens-serif`.

**Pull request closed without merging**